artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/shared/components/login/login.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, inject, OnInit } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { GoogleSigninButtonModule } from "@abacritt/angularx-social-login";
+import { AuthorizationService } from '../../../features/authorization/services/authorization-service';
+import { MaterialModule } from '../../material.module';
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [CommonModule, GoogleSigninButtonModule, MaterialModule],
+  templateUrl: 'login.component.html',
+})
+export class LoginComponent {
+
+  private authService = inject(AuthorizationService);
+
+  userData = this.authService.userData;
+
+  logout()
```

Comment:
troche takie nie jsowe formatowanie? jak ci się chce to możesz prettiera odpalić a jak nie to wyjebongo

Reply:
nie klikłem shift + alt + f